packages/headless/src/app/ssr-engine/types/build.ts

```diff
@@ -26,23 +21,10 @@ export interface BuildWithProps<
   /**
    * Initializes an engine and controllers from the definition.
    */
-  build(
```

Comment:
I extracted method names out of their objects, which will be useful in the next PR. I think it also looks kinda neat.